api/src/main/java/org/apache/iceberg/types/Types.java

```diff
@@ -412,6 +413,24 @@ public String toString() {
     }
   }
 
+  public static class NullType extends PrimitiveType {
```

Comment:
@Ppei-Wang `Null` is not an Iceberg type.  `Unknown` was added as part of v3 spec, but this needs to be handled carefully.

I don't think we can introduce the way you've outlined here.

Reply:
@danielcweeks I understand what you mean.
Could you give me a more detailed introduction to the background of the above problem? As one of the standard specifications for data lake solutions, our company wants to migrate hive tables and views to iceberg to unify management and obtain iceberg's advantages in transaction support, real-time performance and data management.
In the test of hive view->iceberg view, we found the above problem, that is, sql("CREATE VIEW viewName AS SELECT id, NULL as col_test FROM tableName") is not allowed to create a view, which involves null type.
Is there any better solution to this problem?

Reply:
@danielcweeks In addition, for not introducing NULL type， could you please give us a detailed background and your concerns?

Reply:
@Ppei-Wang this would entail extending the [Iceberg table specification](https://iceberg.apache.org/spec/), by adding a new NULL type. Hive might have a NullType, but other query engines might not, so we need to make sure that it is adopted correctly. For changes to the specification, we should have a discussion in the community, it should go through a [Iceberg improvement proposal](https://iceberg.apache.org/contribute/#apache-iceberg-improvement-proposals).

Reply:
@Fokko  ok, Thanks for your suggestion.